easybuild/easyconfigs/l/libxml2/libxml2-2.9.2-GCC-4.8.4.eb

```diff
@@ -0,0 +1,20 @@
+name = 'libxml2'
+version = '2.9.2'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and 
+toolchain developed for the Gnome project
+ (but usable outside of the Gnome platform)."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.4'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
```

Comment:
i've got an issue here - dunno yet if it matters I'm trying on centos7;
i'm with the impression that a python dependency here is missing (or something like that):
```
[fgeorgatos@demo2 ~]$ time eb libxml2-2.9.2-GCC-4.8.4.eb --from-pr 3586 -r
== temporary log file in case of crash /tmp/eb-wcyjid/easybuild-DbBfvo.log
== resolving dependencies ...
== processing EasyBuild easyconfig /tmp/eb-wcyjid/files_pr3586/libxml2/libxml2-2.9.2-GCC-4.8.4.eb
== building and installing libxml2/2.9.2-GCC-4.8.4...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== FAILED: Installation ended unsuccessfully (build directory: /home/fgeorgatos/.local/easybuild/build/libxml2/2.9.2/GCC-4.8.4): build failed (first 300 chars): Python module not loaded
== Results of the build can be found in the log file(s) /tmp/eb-wcyjid/easybuild-libxml2-2.9.2-20160921.174009.RxfhG.log
ERROR: Build of /tmp/eb-wcyjid/files_pr3586/libxml2/libxml2-2.9.2-GCC-4.8.4.eb failed (err: 'build failed (first 300 chars): Python module not loaded')

real    0m7.169s
user    0m3.328s
sys     0m1.037s
```

p.s. despite that, bare python built just fine over here, no issues with: `ml Python/2.7.9-GCC-4.8.4-bare`